doc/tutorials/basic/build_crossfilter_dashboard.md

```diff
@@ -0,0 +1,230 @@
+    "Siemens Gamesa Renewable Energy": "Siemens Gamesa",
+}
+```
+
+Here, we import HoloViews (`hv`) for creating interactive plots, NumPy (`np`) and Pandas (`pd`) for data manipulation, and Panel (`pn`) for dashboard creation. The `NumeralTickFormatter` from Bokeh is used to format numerical values on the plot axes for better readability.
+
+### Data Preparation
+
+The `get_data` function fetches the wind turbine dataset from a URL and preprocesses it by shortening the names of certain manufacturers for clarity in the visualizations.
+
+```python
+@pn.cache()
+def get_data():
+    data = pd.read_csv("https://assets.holoviz.org/panel/tutorials/turbines.csv.gz")
+
+    mask = data.t_manu.isin(list(SHORT_NAMES))
+    data.loc[mask, "t_manu"] = data.loc[mask, "t_manu"].map(SHORT_NAMES)
+    data.t_cap = data.t_cap / 10**6 # Convert capacity to gigawatts for readability
+    return data
+```
+
+`@pn.cache()` decorates the function to cache its output, reducing loading times by avoiding repeated data fetching and processing.
+
+### Creating the Crossfiltering Plot
+
+The core of the dashboard is the `get_plots` function, which generates two interactive bar charts: one displaying the total capacity of wind turbines by year and another by manufacturer.
+
+```python
+@pn.cache
+def get_plots():
+    data = get_data()
+
+    # Define shared dataset
+    ds = hv.Dataset(data, ["t_manu", "p_year", "t_cap"], "t_cap")
+```
+
+The function starts by calling `get_data()` to retrieve the preprocessed dataset. It then uses HoloViews to define a dataset (`ds`) that encapsulates our data, specifying columns for manufacturers (`t_manu`), production year (`p_year`), and turbine capacity (`t_cap`).
+
+:::{note}
+
+In order for [HoloViews linked brushing](https://holoviews.org/user_guide/Linked_Brushing.html) (crossfiltering) to work the plots need to be generated from a shared dataset.
+
+:::
+
+#### Visualizing Data by Year and Manufacturer
+
+Next, we aggregate the data by year and manufacturer to create two separate plots. The plots are formatted to be responsive and use an accent color for consistency.
+
+```python
+    # Create plots
+    ds_by_year = ds.aggregate("p_year", function=np.sum).sort("p_year")
+    ds_by_manufacturer = ds.aggregate("t_manu", function=np.sum).sort(
+        "t_cap", reverse=True
+    )
+    formatter = NumeralTickFormatter(format="0,0")
+    plot_by_year = hv.Bars(
+        ds_by_year, ("p_year", "Year"), ("t_cap", "Capacity (GW)")
+    ).opts(
+        responsive=True,
+        xrotation=90,
+        yformatter=formatter,
+        color=ACCENT,
+        tools=["hover"],
+        active_tools=["box_select"],
+    )
+    plot_by_manufacturer = hv.Bars(
+        ds_by_manufacturer, ("t_manu", "Manufacturer"), ("t_cap", "Capacity (GW)")
+    ).opts(
+        responsive=True,
+        xrotation=90,
+        yformatter=formatter,
+        color=ACCENT,
+        tools=["hover"],
+        active_tools=["box_select"],
```

Comment:
The active_tools don't work fully because of https://github.com/holoviz/holoviews/issues/5119#issuecomment-2030090337.

I would really have liked to only show the `box_select` as rest of tools don't make a lot of sense.

![image](https://github.com/holoviz/panel/assets/42288570/0969fad9-464a-4868-bfb4-44c893b1ce39)

Reply:
This will be fixed upstream.